### DISCUSSION TITLE
There is a bug in Kimai timesheet filtering.

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#5076] [General]
Hello.

[
![스크린샷 2024-09-23 145109](https://github.com/user-attachments/assets/f8482364-e99e-4240-81bf-1263f844cde0)
](url)

When Activity filters timesheets, it only shows the timesheets for that activity, so far it's normal.
But if you click the filtering icon at the top left, click the Find button, and click Retrospective Filtering in that project, it does not reset, and it looks the same as the previous filtering.
If you click the filtering icon in the upper left corner, the previously found filtering settings will remain the same. You won't see the timesheets of other projects or the timesheets of those projects.

When filtering timesheets with a project, I think the activity of filtering should be modified to initialize.

This issue was confirmed in the demo.

Please check.

**Comment by kevinpapst** [comment]
Don't ping me, just to delete that comment afterwards (I get emails for that). Really. I do not want that. Stop it. Please. I told you before.

This is a bug. Please open a new issue

**Comment by kevinpapst** [comment]
Follow up: https://github.com/kimai/kimai/issues/5077
